internals/utils/package.json

```diff
@@ -1,17 +1,134 @@
+    "@sindresorhus/slugify": "0.11.0",
+    "@tootallnate/once": "1.1.2",
+    "@vercel-internals/types": "*",
+    "@vercel/build-utils": "*",
+    "@vercel/client": "*",
+    "@vercel/error-utils": "*",
+    "@vercel/frameworks": "*",
+    "@vercel/fs-detectors": "*",
+    "@vercel/fun": "1.0.4",
+    "@vercel/routing-utils": "*",
+    "ajv": "6.12.2",
+    "ansi-escapes": "4.3.2",
+    "arg": "5.0.0",
+    "async-listen": "1.2.0",
+    "async-retry": "1.1.3",
+    "bytes": "3.0.0",
+    "chalk": "4.1.0",
+    "chokidar": "3.3.1",
+    "date-fns": "1.29.0",
+    "dotenv": "4.0.0",
+    "escape-html": "1.0.3",
+    "fast-deep-equal": "3.1.3",
+    "fs-extra": "10.0.0",
+    "get-port": "5.1.1",
+    "git-last-commit": "1.0.1",
+    "http-proxy": "1.18.1",
+    "ini": "3.0.0",
+    "inquirer": "7.0.4",
+    "is-docker": "2.2.1",
+    "is-port-reachable": "3.1.0",
+    "jsonlines": "0.1.1",
+    "load-json-file": "3.0.0",
+    "mime-types": "2.1.24",
+    "minimatch": "3.0.4",
+    "ms": "2.1.2",
+    "node-fetch": "2.6.7",
+    "npm-package-arg": "6.1.0",
+    "open": "8.4.0",
+    "ora": "3.4.0",
+    "pcre-to-regexp": "1.0.0",
+    "pluralize": "7.0.0",
+    "promisepipe": "3.0.0",
+    "psl": "1.1.31",
+    "raw-body": "2.4.1",
+    "semver": "5.5.0",
+    "serve-handler": "6.1.1",
+    "strip-ansi": "6.0.1",
+    "supports-hyperlinks": "2.2.0",
+    "text-table": "0.2.0",
+    "title": "3.4.1",
+    "tree-kill": "1.2.2",
+    "universal-analytics": "0.4.20",
+    "write-json-file": "2.2.0",
+    "xdg-app-paths": "5.1.0",
+    "yauzl-promise": "2.1.3"
   },
   "devDependencies": {
+    "@types/async-retry": "1.2.1",
+    "@types/bytes": "3.0.0",
+    "@types/chance": "1.1.3",
+    "@types/dotenv": "6.1.1",
+    "@types/escape-html": "0.0.20",
+    "@types/express": "4.17.13",
+    "@types/fs-extra": "9.0.13",
+    "@types/http-proxy": "1.16.2",
+    "@types/ini": "1.3.31",
+    "@types/inquirer": "7.3.1",
+    "@types/jest": "27.4.1",
+    "@types/jsonlines": "0.1.2",
+    "@types/load-json-file": "2.0.7",
+    "@types/mime-types": "2.1.0",
+    "@types/minimatch": "3.0.3",
+    "@types/ms": "0.7.30",
+    "@types/node": "14.18.33",
+    "@types/node-fetch": "2.5.10",
+    "@types/npm-package-arg": "6.1.0",
+    "@types/pluralize": "0.0.29",
+    "@types/psl": "1.1.0",
+    "@types/semver": "6.0.1",
+    "@types/serve-handler": "6.1.1",
+    "@types/text-table": "0.2.0",
+    "@types/title": "3.4.1",
+    "@types/universal-analytics": "0.4.2",
+    "@types/write-json-file": "2.2.1",
+    "@types/yauzl-promise": "2.1.0",
     "@vercel-internals/tsconfig": "*",
+    "@vercel/next": "*",
+    "@vercel/node": "*",
     "@vercel/style-guide": "4.0.2",
+    "chance": "1.1.7",
+    "execa": "3.2.0",
+    "expect": "29.5.0",
+    "express": "4.17.1",
+    "find-up": "4.1.0",
+    "jest-matcher-utils": "29.3.1",
+    "line-async-iterator": "3.0.0",
+    "tmp-promise": "1.0.3",
+    "ts-node": "10.9.1",
     "typescript": "4.9.4"
+  },
+  "jest": {
```

Comment:
Lets use `jest.config.js` instead of package.json which is what most of our packages currently use

Reply:
😅 not `packages/cli` though! This was directly copied over. Do we wanna change that up in this PR?

Reply:
I'm okay either way. It could be a follow up PR if you prefer.

Reply:
i prefer follow up. doing everything I can to keep this pr as minimal changes from what exists as possible